.github/templates/tilgangsinfo-pnd.md

```diff
@@ -33,10 +37,48 @@ labels: 'kind/deploy_patch, team/tilgangsinfo'
 - [ ] [Authentication](https://github.com/Altinn/altinn-authentication).
 - [ ] [Authentication UI](https://github.com/Altinn/altinn-authentication-frontend).
 - [ ] [Resource Registry](https://github.com/Altinn/altinn-resource-registry).
+- [ ] [Auditlog](https://github.com/Altinn/altinn-auth-audit-log)
 
 [Authentication]: https://dev.azure.com/brreg/altinn-studio/_release?_a=releases&view=all&definitionId=20
 [Resource Registry]: https://dev.azure.com/brreg/altinn-studio/_release?_a=releases&view=all&definitionId=36
 
 [^1]: Approve pending prod releases by clicking the blue production chip and clicking approve. ![image-20240306133832594](https://raw.githubusercontent.com/Altinn/altinn-authorization/main/.github/images/ado-pending-approval-screen.png)
 [^2]: Look for any blue production chips. ![image-20240306133137061](https://raw.githubusercontent.com/Altinn/altinn-authorization/main/.github/images/ado-prod-button.png)
+
 [^3]: This is done after checking that everything is running as it should.
+
+[^4]: Check if there are any awaiting deployments to production [here](https://github.com/Altinn/altinn-auth-audit-log/actions/workflows/deploy-after-release.yml)
```

Comment:
These are not valid footnotes.

Reply:
when you mention not valid, do you mean these details are to be described as a paragraph instead of a footnote or do you mean some syntaz feil on the footnotes